test_config.yaml

```diff
@@ -35,11 +35,11 @@
     - proteomics_tmt:
         files:
           - name: agora_proteomics_tmt
-            id: syn32188234
+            id: syn35221005
             format: csv
         final_format: json
         provenance:
-          - syn32188234
+          - syn35221005
         column_rename:
```

Comment:
see comment on config.yaml here

Reply:
This doesn't need to be altered for the test_config.yaml, right?

Reply:
nevermind